vscode/src/services/tree-views/commands.ts

```diff
@@ -0,0 +1,37 @@
+import * as vscode from 'vscode'
+import { CodyTreeItem } from './TreeItemProvider'
+import { CodyCommandMenuItems } from '../../commands'
+import type { CodyCommand } from '@sourcegraph/cody-shared'
+
+/**
+ * Method to get items for the Commands sidebar
+ */
+export function getCommandTreeItems(customCommands: CodyCommand[]): CodyTreeItem[] {
+    const treeNodes = []
+
+    // Create a CodyTreeItem for each group and add to treeNodes
+    for (const item of CodyCommandMenuItems) {
+        const treeItem = new CodyTreeItem(item.key, item.description, item.icon, item.command)
+        treeItem.description = item.keybinding
+
+        if (item.key === 'custom' && customCommands?.length) {
+            treeItem.collapsibleState = vscode.TreeItemCollapsibleState.Expanded
```

Comment:
I think given that we're not persisting the collapse/expand state, like the file explorer in VS Code does, it may be safer to always start this as collapsed, in case people in a shared codebase with lots of commands don't want it to always be expanded.

```suggestion
            treeItem.collapsibleState = vscode.TreeItemCollapsibleState.Collapsed
```

I think ideally we keep it expanded by default if there are commands, but persist/remember if they collapsed it similar to file explorer.

Reply:
If it was collapsed manually by the user, it will stay collapsed until they expanded it again like how it works in the file explorer--is that what you're referring to? 
We won't be able to expand it programmatically again once the user has clicked on the collapse button so I think it'd be nice to start it as expanded to let users know about this new field, but happy to change it in a follow up pr if you think we should start the menu as collapsed :) or if we want to start an a/b test for this just lmk! We actually are seeing a lot of people clicking on the New Chat button on the sidebar!

Reply:
Tl;dr the state is persisted I believe

Reply:
Oh! For some reason it wasn’t for me, but if it is, that’s great 👍🏼